packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -1213,8 +1213,9 @@ export class BasicCrawler<Context extends CrawlingContext = BasicCrawlingContext
                     ...options,
                 });
             },
-            pushData: async (...args: Parameters<Dataset['pushData']>) => {
-                return this.pushData(...args);
+            addRequests: this.addRequests.bind(this),
+            pushData: async (data: Parameters<Dataset['pushData']>[0], datasetIdOrName?: string) => {
```

Comment:
Maybe we can also turn this into `this.pushData.bind(this)`?

Reply:
Hm, yes, turns out we can :slightly_smiling_face: